src/patterns/data/default/entity_attribute.tsv

```diff
@@ -4167,6 +4167,22 @@ OBA:2040170		GO:0030431	sleep	PATO:0000001	quality
 OBA:2040171	sleep duration trait	GO:0030431	sleep	PATO:0001309	duration
 OBA:2040172		GO:0030431	sleep	PATO:0001005	latency
 OBA:2040173		GO:0030431	sleep	PATO:0002325	onset quality
+OBA:2050100	telomere length	GO:0000781	chromosome, telomeric region	PATO:0000122	length
```

Comment:
```suggestion
```
There is already a term OBA:0003784 that has the same logical definition, so please drop OBA:2050100 from the table.